EngineLayer/MetaMorpheusEngine.cs

```diff
@@ -172,19 +238,21 @@ public static double CalculateClassicScore(IMsDataScan<IMzSpectrum<IMzPeak>> thi
                     MatchingProductsHere++;
                     MatchingIntensityHere += experimental_intensities[experimentalIndex];
 
-                    currentTheoreticalIndex++;
+                    currentTheoreticalIndex++; //prevent multi counting
                     if (currentTheoreticalIndex == TotalProductsHere)
                         break;
-                    currentTheoreticalMz = sortedTheoreticalProductMassesForThisPeptide[currentTheoreticalIndex] + Constants.protonMass;
+                    currentTheoreticalMass = sortedTheoreticalProductMassesForThisPeptide[currentTheoreticalIndex];
+                    currentTheoreticalMz = currentTheoreticalMass + Constants.protonMass;
                 }
-                // Else if for sure passed a theoretical
+                // Else if for sure did not reach the next theoretical yet
                 else if (currentExperimentalMz > currentTheoreticalMz)
                 {
                     // Move on to next index and never come back!
                     currentTheoreticalIndex++;
                     if (currentTheoreticalIndex == TotalProductsHere)
                         break;
-                    currentTheoreticalMz = sortedTheoreticalProductMassesForThisPeptide[currentTheoreticalIndex] + Constants.protonMass;
+                    currentTheoreticalMass = sortedTheoreticalProductMassesForThisPeptide[currentTheoreticalIndex];
```

Comment:
These masses are removed on purpose! For efficiency, no need to bring them back

Reply:
Didn't realize that change had been made; I was using the old code. Thanks for letting me know.